### ISSUE TITLE
Username already exists/Does nothing

### ISSUE THREAD

**Issue by serberus7** [jepayne1138/PTCAccount#14]
Was working for the first few times
Now it says the username already exists on everything if i specify one
OR!
If i don't specify any paramaters and it goes to make a random account, it just sits there and does nothing.

**Comment by serberus7** [comment]
NOOOO
Captcha at the bottom of account creation page now.
I got to use this wonderful script only 3 times :(

**Comment by scalp42** [comment]
Yeah the party is over, they added a Captcha at the bottom, can confirm.

**Comment by jepayne1138** [comment]
Well darn. Can't really say it's unexpected. Looks like the party's over.

**Comment by serberus7** [comment]
nothing we can do about a captcha?

**Comment by serberus7** [comment]
no way at all? :(

**Comment by subzerofun** [comment]
can´t we just ask some image classification experts very nicely to help us? :-)
in all honesty i don´t think that a bot can solve the google captchas. and even if it would be possible – the solving part would require a lot of computational power. they also track the movement of your cursor, check if it´s irregular – human. so the bot would have to mimic that too...

**Comment by subzerofun** [comment]
more info:
https://www.blackhat.com/docs/asia-16/materials/asia-16-Sivakorn-Im-Not-a-Human-Breaking-the-Google-reCAPTCHA-wp.pdf
